volume-providers/vpc/vpcclient/instances/fakes/volume_attach_service.go

```diff
@@ -263,65 +263,65 @@ func (fake *VolumeAttachService) GetVolumeAttachmentReturnsOnCall(i int, result1
 	}{result1, result2}
 }
 
-func (fake *VolumeAttachService) ListVolumeAttachment(arg1 *models.VolumeAttachment, arg2 *zap.Logger) (*models.VolumeAttachmentList, error) {
-	fake.listVolumeAttachmentMutex.Lock()
-	ret, specificReturn := fake.listVolumeAttachmentReturnsOnCall[len(fake.listVolumeAttachmentArgsForCall)]
-	fake.listVolumeAttachmentArgsForCall = append(fake.listVolumeAttachmentArgsForCall, struct {
+func (fake *VolumeAttachService) ListVolumeAttachments(arg1 *models.VolumeAttachment, arg2 *zap.Logger) (*models.VolumeAttachmentList, error) {
```

Comment:
actually method name either it should be `ListVolumeAttachment` or `VolumeAttachments` I think previous one looks good, same thing applicable for other places as well